oxidation/crates/parsec_client_types/src/local_manifest.rs

```diff
@@ -111,6 +183,99 @@ impl From<LocalFileManifest> for LocalFileManifestData {
 
 impl_local_manifest_dump_load!(LocalFileManifest);
 
+impl LocalFileManifest {
+    pub fn get_chunks(&self, block: usize) -> Vec<Chunk> {
```

Comment:
the return value should be `Vec<&Chunk>` considering how get_chunks is used in the python codebase